stories/core/ToggleButton.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+import { action } from '@storybook/addon-actions'
+import { ToggleButton } from 'react-ui-core/src/ToggleButton'
+import StoryBookTheme from '../theme/Storybook.css'
+
+export const DefaultToggleButton = (
+  <ToggleButton
+    className={classnames(
+      StoryBookTheme['Story-center'],
+    )}
+    onClick={value => {
+      action('The button is ')(`${(value) ? 'toggled' : 'untoggled'}`)
+    }}
+  >
+    <span className="DefaultToggleButt" >Default Title</span>
+  </ToggleButton>
+)
+
+export const ToggleSVG = (
```

Comment:
This isn't really any different than the first story.  How about we just use this one instead?

Reply:
Removed